Azure/azure-cli
# [Keyvault] `az keyvault key rotation-policy update`: Support both camel case and snake case json for `--value`

**Description**

Previously, `az keyvault key rotation-policy update` only accept policy like this:
```
{
  "lifetime_actions": [
    {
      "trigger": {
        "time_after_create": "P15D",
        "time_before_expiry" : null
      },
      "action": {
        "type": "Rotate"
      }
    },
    {
      "trigger": {
        "time_before_expiry" : "P7D"
      },
      "action": {
        "type": "Notify"
      }
    }
  ],
  "attributes": {
    "expires_in": "P30D"
  }
}
```

This PR add support for policy json used by ARM template:
```
{
  "lifetimeActions":[
	{
  	  "trigger":{"timeAfterCreate":"P18M"},
 	  "action":{"type":"rotate"}
	},
	{
	  "trigger":{"timeBeforeExpiry":"P30D"},
	  "action":{"type":"notify"}
	}
  ],
  "attributes":{
	"expiryTime":"P2Y"
  },
}
```
and policy json returned by Python SDK:
```
{
  "expiresIn": "P2Y",
  "lifetimeActions": [
    {
      "action": "Rotate",
      "timeAfterCreate": "P18M",
      "timeBeforeExpiry": null
    },
    {
      "action": "Notify",
      "timeAfterCreate": null,
      "timeBeforeExpiry": "P30D"
    }
  ]
}
```
**Testing Guide**

`az keyvault key rotation-policy --value /path/to/policy.json --name key_name --vault-name vault_name`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
